pkgs/applications/networking/mailreaders/notmuch/default.nix

```diff
@@ -90,6 +91,8 @@ stdenv.mkDerivation rec {
 
   postInstall = stdenv.lib.optionalString withEmacs ''
     moveToOutput bin/notmuch-emacs-mua $emacs
+  '' + optionalString withVim ''
+    make -C vim DESTDIR="$out/share/vim" prefix="" install
```

Comment:
Is this path working for you? I am fairly sure that it'd be safer to do the following:

```sh
make -C vim DESTDIR="$out/share/vim-plugins/notmuch" prefix="" install
```

Also for Neovim users, it'd be nice if you'd also add here:

```sh
mkdir -p $out/share/nvim
ln -s $out/share/vim-plugins/notmuch $out/share/nvim/site
```